foundry.toml

```diff
@@ -9,6 +9,10 @@ optimizer_runs = 20000
 test = 'test'
 solc = '0.8.26'
 fs_permissions = [{ access = 'read-write', path = './deploy-out' }, { access = 'read', path = './out' }]
+##### Uncomment if running storage layout verification
+ffi = true
+ast = true
+build_info = true
```

Comment:
Should consider to comment them by default? since it would disable incremental build, would make local build/test slow.

Reply:
Sure, we need to figure it out how to introduce it for CI/CD after mainnet, to make sure we don't write storage layout problems.